schema/mysql/schema.sql

```diff
@@ -1013,12 +1013,12 @@ CREATE TABLE acknowledgement_history (
 
   set_time bigint unsigned NOT NULL,
   clear_time bigint unsigned DEFAULT NULL,
-  author varchar(255) NOT NULL COLLATE utf8mb4_unicode_ci,
+  author varchar(255) DEFAULT NULL COLLATE utf8mb4_unicode_ci COMMENT 'NULL if ack_set event happened before Icinga DB history recording',
   cleared_by varchar(255) DEFAULT NULL COLLATE utf8mb4_unicode_ci,
   comment text DEFAULT NULL,
```

Comment:
I would add the NULL info here too, as a comment is actually required.

Reply:
Any idea why this column was already NULLable?

Reply:
Nope, no idea. Maybe you? Since you've introduced this table 😆